### DISCUSSION TITLE
GitVersion - include ShortSha

### DISCUSSION THREAD

**Discussion by huw-dhcw** [cake-build/cake#3915] [Extension Q&A]
Can we have the ShortSha property added to the returned GitVersion object in the GitVersion tool please?

**Comment by nils-a** [comment]
Have you seen that there is `GitShortenSha()` [available](https://cakebuild.net/api/Cake.Git/GitAliases/FF333F40) [since *Cake.Git* version 2.0.0](https://github.com/cake-contrib/Cake_Git/releases/tag/v2.0.0)?

**Reply by huw-dhcw** [reply]
Yes, but TBH for what I wanted it was easier for me to just take the first 7 chars!
I raised this issue for completeness as GitVersion already does the hard work, it just requires a property to map into.
I may even submit a PR for it myself if I have time.

**Reply by nils-a** [reply]
I created https://github.com/cake-build/cake/issues/3916 to track that.
